pkg/prometheus/operator.go

```diff
@@ -624,8 +625,9 @@ func (c *Operator) syncNodeEndpoints(ctx context.Context) error {
 		ObjectMeta: metav1.ObjectMeta{
 			Name: c.kubeletObjectName,
 			Labels: c.config.Labels.Merge(map[string]string{
-				"k8s-app":                "kubelet",
-				"app.kubernetes.io/name": "kubelet",
+				"k8s-app":                   "kubelet",
+				"app.kubernetes.io/name":    "kubelet",
+				"app.kubernetes.io/part-of": "prometheus-operator",
```

Comment:
Shouldn't this be `app.kubernetes.io/managed-by` as prometheus-operator manages this resource, but this resource isn't a part of prometheus-operator?

Reply:
Was thinking about it as well (thus why my branch is called managed-by ;-) ). I figured that part-of for the endpoint is a bette rmatch. Open to change it back.

Reply:
I think we should have a `managed-by` one as this is certain that object is managed by prometheus-operator. However in case of `part-of` I am not so sure, on one hand this endpoint is a part of kubelet as data exposed over it are coming from kubelet. On the other hand, kubelet doesn't require this object so it makes more sense to say it is a part of prometheus-operator.

That said I think we should have either only `managed-by` or both.

Reply:
Okay, I switched it back to `managed-by`